inference/server/oasst_inference_server/user_chat_repository.py

```diff
@@ -1,33 +1,34 @@
 import fastapi
+import pydantic
 import sqlmodel
 from loguru import logger
 from oasst_inference_server import models
 from oasst_shared.schemas import inference
 
 
-class UserChatRepository:
-    def __init__(self, session: sqlmodel.Session, user_id: str | None, do_commit=True) -> None:
-        self.session = session
-        self.user_id = user_id
-        self.do_commit = do_commit
+class UserChatRepository(pydantic.BaseModel):
```

Comment:
What's the rationale for making this a `BaseModel` here? Should we not have this be consistent with `ChatRepository` and either have both or neither as `BaseModel`s?

Reply:
the only rationale is to trigger pydantic validation, i.e. make sure `user_id` is really a string, but we could also do that ourselves